.evergreen/config.yml

```diff
@@ -490,7 +490,6 @@ functions:
     - command: shell.exec
       type: test
       params:
-        silent: true
         working_dir: "src"
         script: |
           # DO NOT ECHO WITH XTRACE (which PREPARE_SHELL does)
```

Comment:
Could you silently load the SASL etc. vars like we do for the "run atlas tests" below? This would avoid the concerns about xtrace and leaking creds.

Reply:
Done